components/DataTable/src/CommunityToolkit.WinUI.Controls.DataTable.csproj

```diff
@@ -6,6 +6,7 @@
 
     <!-- Rns suffix is required for namespaces shared across projects. See https://github.com/CommunityToolkit/Labs-Windows/issues/152 -->
     <RootNamespace>CommunityToolkit.WinUI.Controls.DataTableRns</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why do we need UnsafeBlocks for these components? This seems odd...

Reply:
It's because CsWinRT needs unsafe code to generate some marshalling stub for generic collections.

Reply:
See also https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/223